scripts/tests/test_bert.py

```diff
@@ -0,0 +1,39 @@
+import os
+import subprocess
+import time
+import mxnet as mx
+
+import pytest
+
+@pytest.mark.serial
+@pytest.mark.remote_required
+def test_pretrain():
```

Comment:
ideally this should go into test_scripts. does it only work with mxnet nightly?

Reply:
Yes